vent/controller/control_module.py

```diff
@@ -813,11 +905,18 @@ def _reset(self):
 
 class ControlModuleSimulator(ControlModuleBase):
     # Implement ControlModuleBase functions
-    def __init__(self):
+    def __init__(self, simulator_dt = None):
```

Comment:
so if you
```python
controller = ControlModuleSimulator(simulator_dt = 0.01)
controller._LOOP_UPDATE_TIME = 0
```
the simulation can be run as fast as possible while still spitting out values that are computed as if `simulator_dt` seconds have passed. timestamps will be wrong though...